webapp/src/components/add_issue/add_issue.jsx

```diff
@@ -77,13 +77,18 @@ export default class AddIssue extends React.PureComponent {
 
     submit = () => {
         const {submit, postID, assignee, closeAddBox, removeAssignee} = this.props;
-        const {message, description, attachToThread} = this.state;
+        const {message, description, attachToThread, sendTo} = this.state;
         this.setState({
             message: '',
             description: '',
         });
+
         if (attachToThread) {
-            submit(message, description, assignee.username, postID);
+            if (assignee) {
+                submit(message, description, assignee.username, postID);
+            } else {
+                submit(message, description, sendTo, postID);
+            }
```

Comment:
What's the difference between assigning a todo to someone and sending it to someone?

Reply:
@mickmister I think one assigns it to another person, while the other is just your own.

Reply:
Not sure what this means. Sending it to someone means you're sending it to yourself? Maybe there's been a fundamental change to how this works semantically?